deploy/kustomize/README.md

```diff
@@ -14,15 +14,15 @@ There are two distinct options to deliver HCO operator to OLM - Marketplace and
 ### Marketplace
 This method is taking advantage of OperatorSource, which makes the operator available on OLM OperatorHub (implicitly creating a CatalogSource with the same name).
 To manually deliver HCO using marketplace, edit `spec.registryNamespace` of `marketplace/operator_source.yaml` to the desired value (default is "kubevirt-hyperconverged"), and run:
-```
-oc apply -k marketplace
+```bash
```

Comment:
such changes are better left for a follow-up PRs as they confuse the poor reviewer ;-)

Reply:
I took the opportunity of this doc PR to make cosmetic fixes, instead of openning a PR just for this and execute the CI jobs for nothing.